Update Docs for native-image Commons Logging (#10474)
quarkusio/quarkus: docs/src/main/asciidoc/logging.adoc

See also: <<getting-started-testing.adoc#test-from-ide,Running `@QuarkusTest` from an IDE>>

[[logging-native-image]]
== Logging with native image

> If you are using libraries that have dependencies on `Apache Commons Logging` and experience complications building the native image such as class not found, you can resolve this by integrating the `JBoss Logging` dependecy.

Reviewer: ```suggestion
If you are using libraries that have dependencies on `Apache Commons Logging` and are experiencing complications building the native image due to missing logging classes, you can resolve this by integrating the `JBoss Logging` dependency.
```

The error seen when building native-image:
```

> Caused by java.lang.ClassNotFoundException: org.apache.commons.logging.impl.LogFactoryImpl)

Reviewer: ```suggestion
Caused by java.lang.ClassNotFoundException: org.apache.commons.logging.impl.LogFactoryImpl
```

See also: <<getting-started-testing.adoc#test-from-ide,Running `@QuarkusTest` from an IDE>>

[[logging-native-image]]
== Logging with native image

If you are using libraries that have dependencies on `Apache Commons Logging` and experience complications building the native image such as class not found, you can resolve this by integrating the `JBoss Logging` dependecy.

The error seen when building native-image:
```
Caused by java.lang.ClassNotFoundException: org.apache.commons.logging.impl.LogFactoryImpl)
```

Resolve by adding the associated `JBoss Logging` dependency:
[source, xml]
----
        <dependency>
            <groupId>org.jboss.logging</groupId>
            <artifactId>commons-logging-jboss-logging</artifactId>

>             <version>1.0.0.Final</version>

Reviewer 2: Hello @oztimpower, Should users be mentioning these versions here? Or does our bom file for Quarkus manage these versions? I think it's better to not have these versions specified here.
